honeybadger-io/honeybadger-js
# feat(rollup): add configurable parallelism to sourcemap upload

## Status
**READY**

## Description
Part of https://github.com/honeybadger-io/honeybadger-js/issues/1013. 
Closes https://github.com/honeybadger-io/honeybadger-js/issues/1011. 

Now that `@honeybadger-io/plugin-core` is published, this PR allows `rollup-plugin` to rely on that package and therefore take advantage of the configurable parallelism available in the `workerCount` option. 

## Related PRs
https://github.com/honeybadger-io/honeybadger-js/pull/1193

## Todos

## Steps to Test or Reproduce

1. Build the rollup plugin (`npm i && npm run build`)
2. Build each of the example projects (`npm run build`)
3. Confirm that sourcemaps were uploaded and deploy notification was sent in the HB UI.
